README.md

```diff
@@ -7,14 +7,45 @@ TypeScript types for the Seam API.
 
 ## Description
 
-TODO
+This repository aggregates the latest route schemas and type definitions for the Seam API.
+Upstream Seam repositories automatically contribute updates on every deployment
+via a Pull Request opened by Seam Bot.
+This ensures downstream consumers, e.g., the SDK, fakes, documentation generators, etc.,
+have a single source of truth that stays up to date with the API.
+
+### Structure
+
+The main entrypoint for this module is empty: there is dedicated entrypoint for each Seam API namespace.
+
+Each directory under `src/lib/seam/*` is owned by an upstream Seam repository.
+These files should not be edited manually as they will be overridden by automation.
+
+Each entrypoint may export one or more of the following:
+
+- Zod schemas: collectively exported as a single object named `schemas`.
+- Types: derived directly from the [Zod Schemas][zod] and exported at the top level.
+- A separate Zod schmea named `routes` generated by [nextlove] and exported at the top level.
+- A type named `Routes` generated by [nextlove].
```

Comment:
I was going to link to the nextlove docs on the structure of this but I think we only document the openapo spec?

Reply:
[typed-axios](https://github.com/seamapi/typed-axios) documents the structure of the route type